Q24ECU/core/include/interfaces/interface_spi.h

```diff
@@ -0,0 +1,170 @@
+
```

Comment:
I honestly think it might be best to move most of this into some sort of a reference file. The only thing in use by the SPI right now is the shift registers. So there won't be any data reception or writes longer than one byte -> no need for interrupts. Maybe just remove the file extension for now and we will revisit this at a later date

Reply:
Changed it to a txt file for now